apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -132,6 +187,63 @@ export default class GroupManagement extends Vue {
     this.editedGroup = null;
   }
 
+  displayChip(width: number, numberOfDisplayedUsers: number, members: ISlimUser[]): boolean {
+    if(window.innerWidth >= width && numberOfDisplayedUsers <= members.length) {
+      return true;
+    }
+    else {
+      return false;
+    }
+  }
+
+  displayTotalMembersChip(numberOfMembers: number): boolean {
+    // If the group has more than 3 members, than show the total chip.
+    if(numberOfMembers > 3) {
+      return true;
+    }
+    else {
+      if(window.innerWidth > 1200) {
+        return false;
+      }
+      else if(window.innerWidth >= 992 && numberOfMembers > 2) {
+        return true;
+      }
+      else if(window.innerWidth >= 0 && numberOfMembers > 1) {
+        return true;
+      }
+      else {
+        return false;
+      }
+    }
+  }
```

Comment:
Don't worry about different screen sizes, it makes this a lot more complex and most people will just be using it on a desktop. It collapses down well enough that I can deal with it displaying 4 chips all the time if it means less logic here.